src/config/index.ts

```diff
@@ -11,4 +11,5 @@ export default {
     release: process.env.GIT_SHA || '',
     environment: process.env.NODE_ENV || 'development',
   },
+  graphQLProxy: process.env.GRAPHQL_PROXY || 'https://getpocket.com/graphql',
```

Comment:
this is pointing to `prod` and we don't have a dev equivalent. 

assuming the accessToken/consumerKey check would fail for random test data - so considering this as low risk